### PR TITLE
Initial Next.js project

### PR DESCRIPTION
**Pull request by 4KevR**
Created a new Next.js project in the frontend folder using the npm package manager

### CODE REVIEW

**Review by memeToasty — APPROVED** [review]
LGTM

**Review by Julian702 — APPROVED** [review]

File: frontend/README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+
+You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
```
Comment (TomRomeo) [inline frontend/README.md:37]: Could you remove those lines?

**Review by TomRomeo — CHANGES_REQUESTED** [review]
Looks great but could you delete the boilerplate code that was added to the frontend readme? :)

File: frontend/README.md (hunk 1 above)
Comment (4KevR) [reply frontend/README.md]: Removed in commit 36e57f8a707355ead8b65ac2c415cd98fb452250

**Review by TomRomeo — APPROVED** [review]

**Pull request merged**